src/disco/css/Addon.scss

```diff
@@ -166,17 +167,17 @@ $addon-padding: 20px;
   }
 
   &:not(.theme) .content {
-    align-items: left;
-    flex-direction: column;
+    align-items: center;
+    flex-direction: row;
     width: calc(100% - 94px);
   }
 
-  @include respond-to(large) {
-    &:not(.theme) .content {
-      align-items: center;
-      flex-direction: row;
-    }
-  }
+  // @include respond-to(large) {
```

Comment:
This should be deleted if we don't need it

Reply:
Shoot, sorry, meant to and forgot. I've been so forgetful with these this week, thanks!